free/src/main/scala/cats/free/FreeApplicative.scala

```diff
@@ -1,49 +1,56 @@
 package cats
 package free
 
+import cats.arrow.NaturalTransformation
+
 /** Applicative Functor for Free */
 sealed abstract class FreeApplicative[F[_], A] { self =>
-
-  import FreeApplicative.{FA, Pure, Ap}
+  import FreeApplicative.{FA, Pure, Ap, ap => apply, lift}
```

Comment:
I think this example would be a little bit more clear without aliasing `ap` to `apply`.

What do you think? If you disagree I am ok to merge this as-is.